bundles/org.openhab.binding.remoteopenhab/src/main/resources/OH-INF/addon/addon.xml

```diff
@@ -6,5 +6,6 @@
 	<type>binding</type>
 	<name>Remote openHAB Binding</name>
 	<description>The Remote openHAB binding allows to communicate with remote openHAB servers.</description>
+	<connection>local</connection>
```

Comment:
This can be discussed.
Yes it is local if your remote server is in the same network as your OH server but it would be cloud and you need an internet access in case you "connect" to a remote OH server not in the same local network.

Reply:
I was about to post a similar comment when first seeing this, but decided not to. It's an interesting case, but also gets kind of philosophical - what does cloud really mean? You could put any device outside your own network, even on the internet, but that doesn't make it cloud. To me "cloud" means "a service on the internet completely outside of my control", but that's probably opinionated. If it just means "a service on the internet", then we are back to the problem that anything can be put on the internet, so how do we distinguish? I could also expose a local service on the internet to a friend, which would be local for me, but "cloud" to him. 🙂

Reply:
> In this case, maybe we should look at the most common use-case.

I would go with that because you can also put your serial devices in the cloud and connect to them using a rfc2217 connection provided by the serial transport. :wink:

Reply:
I believe a common case could be that a user owns a secondary house and wants to monitor it with its main house.

Could we exclude the remote openHAB binding from this PR ?

Reply:
Let's figure that one out another time. :slightly_smiling_face: